packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.kt

```diff
@@ -82,6 +82,137 @@ open class FileDownloader(context: Context) {
     )
   }
 
+  internal fun parseManifestResponse(response: Response, configuration: UpdatesConfiguration, callback: ManifestDownloadCallback) {
+    val contentType = response.header("content-type") ?: ""
+    val regex = Regex("multipart/.*boundary=\"?([^\"]+)\"?")
```

Comment:
Seems like this could break if there are no double quotes, which are treated as optional anyway. `[^\"]` will consume the rest of the string in that case.

Probably should use backreferences too like `(\"?)....(\\1)` or have a little parser, which is needed to parse content-disposition too.